### PR TITLE
[System.Xml.Private] Extracts XmlSchemaConstants

### PR DESCRIPTION
**Pull request by marek-safar**
- Moved to new file
- Removed unused constant
- Moved to assembly namespace

### CODE REVIEW

File: src/System.Private.Xml/src/System.Private.Xml.csproj [hunk 1]
```diff
@@ -346,6 +346,7 @@
     <Compile Include="System\Xml\Schema\XmlSchemaComplexContentExtension.cs" />
     <Compile Include="System\Xml\Schema\XmlSchemaComplexContentRestriction.cs" />
     <Compile Include="System\Xml\Schema\XmlSchemaComplexType.cs" />
+    <Compile Include="System\Xml\Schema\XmlSchemaConstants.cs" />
```
Comment (karelz) [inline src/System.Private.Xml/src/System.Private.Xml.csproj]: You put the file into Serialization directory.

Comment (marek-safar) [inline src/System.Private.Xml/src/System.Private.Xml.csproj]: Oups, fixed the file location

**Comment by marek-safar** [comment]
We are trying to see/test how usable is NET_NATIVE implementation of XML serialization for Xamarin products

**Comment by karelz** [comment]
> We are trying to see/test how usable is NET_NATIVE implementation of XML serialization for Xamarin products

Do you know if it already shipped in .NET Native? Or is it part of .NET Standard 2.0?
If it is the former, then I expect it should mostly work, unless we broke it by accident during our .NET Standard 2.0 effort.
If it is the latter, then it will be part of our .NET Standard 2.0 UWP effort (plan & schedule is under discussion now) - you might want to sync up & coordinate with @sepidehMS @krwq and @danmosemsft in such case.

**Comment by marek-safar** [comment]
I don't know but when we ran it with Mono tests we found few notimplemented features and several bugs hence I am guessing it didn't ship. I don't know where is the latest code being developed but I thought it's under CoreFX repo

**Comment by karelz** [comment]
> I don't know where is the latest code being developed but I thought it's under CoreFX repo

Latest BCL source code is in CoreFX repo (technically lower-level APIs are in CoreCLR & CoreRT repos).

**Comment by marek-safar** [comment]
Is there anything else for me to do?

**Comment by karelz** [comment]
@sepidehMS @krwq do you want to review it before merge?

**Pull request merged**